tests/test_noise_models.py

```diff
@@ -0,0 +1,230 @@
+                    wires=[0, 1],
+                ),
+            ),
+            (
+                noise.phase_amplitude_damping_error(0.3451, 0.2356),
+                qml.QubitChannel(
+                    np.array(
+                        [
+                            [[-0.97035755 + 0.0j, 0.0 + 0.0j], [0.0 + 0.0j, -0.74564448 + 0.0j]],
+                            [[-0.2416738 + 0.0j, 0.0 + 0.0j], [0.0 + 0.0j, 0.31450645 + 0.0j]],
+                            [[0.0 + 0.0j, 0.58745213 + 0.0j], [0.0 + 0.0j, 0.0 + 0.0j]],
+                        ]
+                    ),
+                    wires=AnyWires,
+                ),
+            ),
+            (
+                noise.kraus_error(
+                    [
+                        [[-0.97035755 + 0.0j, 0.0 + 0.0j], [0.0 + 0.0j, -0.74564448 + 0.0j]],
+                        [[-0.2416738 + 0.0j, 0.0 + 0.0j], [0.0 + 0.0j, 0.31450645 + 0.0j]],
+                        [[0.0 + 0.0j, 0.58745213 + 0.0j], [0.0 + 0.0j, 0.0 + 0.0j]],
+                    ],
+                ),
+                qml.QubitChannel(
+                    np.array(
+                        [
+                            [[-0.97035755 + 0.0j, 0.0 + 0.0j], [0.0 + 0.0j, -0.74564448 + 0.0j]],
+                            [[-0.2416738 + 0.0j, 0.0 + 0.0j], [0.0 + 0.0j, 0.31450645 + 0.0j]],
+                            [[0.0 + 0.0j, 0.58745213 + 0.0j], [0.0 + 0.0j, 0.0 + 0.0j]],
+                        ]
+                    ),
+                    wires=AnyWires,
+                ),
+            ),
+        ],
+    )
+    def test_build_kraus_error_ops(self, qiskit_error, pl_channel):
+        """Tests that a quantum error can be correctly converted into a PennyLane QubitChannel."""
+        pl_op_from_qiskit = _build_qerror_op(qiskit_error)
+        choi_mat1 = self._kraus_to_choi(
+            pl_op_from_qiskit.compute_kraus_matrices(*pl_op_from_qiskit.data)
+        )
+        choi_mat2 = self._kraus_to_choi(pl_channel.compute_kraus_matrices(*pl_channel.data))
+        assert np.allclose(choi_mat1, choi_mat2)
+
+    @pytest.mark.parametrize(
+        "depol1, depol2, exc_pop",
+        [
+            (0.123, 0.456, 0.414),
+            (0.631, 0.729, 0.128),
+            (0.384, 0.657, 0.902),
+        ],
+    )
+    def test_build_model_map(self, depol1, depol2, exc_pop):
+        """Tests that _build_noise_model_map constructs correct model map for a noise model"""
+        error_1 = noise.depolarizing_error(depol1, 1)
+        error_2 = noise.depolarizing_error(depol2, 2)
+        error_3 = noise.phase_amplitude_damping_error(0.14, 0.24, excited_state_population=exc_pop)
+
+        noise_model = noise.NoiseModel()
+        noise_model.add_all_qubit_quantum_error(error_1, ["rz", "sx", "x"])
+        noise_model.add_all_qubit_quantum_error(error_2, ["cx"])
+        noise_model.add_all_qubit_quantum_error(error_3, ["ry", "rx"])
+
+        model_map, _ = _build_noise_model_map(noise_model)
```

Comment:
```suggestion
        quantum_error_model_map, _ = _build_noise_model_map(noise_model)
```

Might be better to stay consistent across files